<launch>
	
	<node pkg="eod_pkg" type="hs_interface" name="hs_interface" required="true">
   </node>

   <node pkg="eod_pkg" type="motor_controller" name="motor_controller" required="true">
   </node>

   <node pkg="eod_pkg" type="odometry" name="odometry" required="true">
   </node>

   <node pkg="eod_pkg" type="ref_transformer" name="ref_transformer" required="true">
   </node>

   <node pkg="eod_pkg" type="wheels_vel_sensor" name="wheels_vel_sensor" required="true">
   </node>

</launch>